### ISSUE TITLE
NuGet update doesn't retain CopyToOutputDirectory in Proj file setting for NLog.config

### ISSUE THREAD

**Issue by TechSavvySam** [NLog/NLog#2608]
**Type** (choose one): Bug

**NLog version**: 4.4.13

**Platform**: .Net 4.5

**Current Result**

The update changed this in the .csproj file:

    <Content Include="NLog.config">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>

to:

    <Content Include="NLog.config">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>

**Expected Result**

It should leave it set to "PreserveNewest" if that's what is already there.

BECAUSE:

When you set it to "Always" that **forces** a rebuild of that project (and all of its dependencies) in VS2017 **every single time** you run your solution even if nothing has changed in the project.  So anyone who has set this to "PreserveNewest" has taken the time to figure this out and doesn't want it changed on every nuget update.

**Comment by 304NotModified** [comment]
> It should leave it set to "PreserveNewest"


Not sure if we could control this

This will only be set after a nuget install, isn't?

Guess you could control this by install nlog and not the nlog.config nuget package

Anyway, "PreserveNewest", is maybe a better default

**Comment by TechSavvySam** [comment]
I just upgraded from 4.4.12 to 4.4.13 and it **changed** it.  I caught this because I had just spent a lot of time two weeks ago tracking down why every one of my projects was rebuilding every time I ran the solution even though nothing had changed.

I don't know anything about nuget updates, so I don't know if it is deleting the entry and re-adding it or not.  If it is, that's probably going to make it harder to fix.  Hopefully it's just updating it and it can decide to leave the setting alone.
